graph.go

```diff
@@ -130,9 +135,13 @@ func Graph(units unit.SourceUnits) (*graph.Output, error) {
 		pkgs = append(pkgs, pkg)
 	}
 
-	o, err := doGraph(pkgs)
-	if err != nil {
-		return nil, err
+	var o gog.Output
+	for _, pkg := range pkgs {
+		output, err := doGraph(pkg)
+		if err != nil {
+			return nil, err
+		}
+		o.Append(output)
 	}
```

Comment:
Yes, this is an option. However we should first see if we want to pull this loop up in a higher level (related to removing `graphAll`).